client/time/time-ago.ts

```diff
@@ -0,0 +1,48 @@
+const MINUTE = 60 // in seconds
+const HOUR = MINUTE * 60
+const DAY = HOUR * 24
+const WEEK = DAY * 7
+const MONTH = DAY * 30.42
+const YEAR = MONTH * 12
+
+/**
+ * A function which takes the number of seconds representing a difference between two dates, and
+ * returns a string representation of that difference, in "Xy ago" syntax. Where X equals amount of
+ * time, and y specifies the time unit.
+ *
+ * Note that this is not a *perfect* measurement of time. Some values are approximated, e.g. number
+ * of days in a month, and leap days/seconds/whatever are not taken into account.
+ */
```

Comment:
For functions where parameters could use some explanation (e.g. you need to explain the format they're expected to be in, the units, etc.), it's good to include `@param` entries in the doc comment